_posts/2019-11-07-Java-Information-Security.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Deploy Java, UIS Security Standards"
+date: 2019-11-07 09:30
+categories: upcoming
+---
+
+## Meta:
+
+- Location: [Walter Library](http://campusmaps.umn.edu/walter-library), Room 402 and on [Zoom](https://z.umn.edu/cpmstream)
+- Day: Thursday, Nov 7
+- Time: 9:30 - 11:00
+
+## Agenda:
+
+- 9:30 - Introductions
+- 9:35 - 10:00 [Use Jib to Deploy Java the Easy Way](#use-jib-to-deploy-java-the-easy-way), Travis Sjobeck - Core Infrastructure Admin, OIT
+- 10:00 - 10:15 Break
+- 10:25 - 10:45 [Updated Information Security Standards and Gap Analysis](#updated-information-security-standards-and-gap-analysis), Ray Phillips - University Information Security, OIT
```

Comment:
The times don't quite line up. Do we want to 1) extend the break for 10 min, 2)move up the UIS talk 10 min or 3) split the difference (extend break by 5 and move UIS up by 5)?

I think we should at least move up the UIS talk a little (options 2 or 3). That will give us a little cushion for lightning talks if their talk goes long.

Reply:
I noticed this too.  I adjusted the times now.  What do you think?